tsconfig.json

```diff
@@ -11,13 +11,13 @@
     "skipLibCheck": true,
     "noImplicitAny": true,
     "noImplicitThis": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
     "isolatedModules": true,
     "esModuleInterop": true,
     "strictNullChecks": true,
     "noImplicitReturns": true,
     "resolveJsonModule": true,
-    "noUnusedParameters": true,
+    "noUnusedParameters": false,
```

Comment:
![but-why](https://user-images.githubusercontent.com/692077/141340263-b285c8f9-cfb5-4f4f-b0ba-53f2b595f886.gif)

After seeing this I've opened `BuyItemSlotsModal.container.ts` and saw that Dispatch is imported and not being used :(

Reply:
Ohh, I removed it to make the development a bit faster but I forgot about re-adding it again.